File: src/ELibrary.Backend/ShopApi/Features/StatisticsFeature/Domain/Models/GetShopStatistics.cs
namespace ShopApi.Features.StatisticsFeature.Domain.Models
{
    public class GetShopStatistics
    {
        public DateTime? FromUTC { get; set; }
        public DateTime? ToUTC { get; set; }
        public StatisticsBook[] IncludeBooks { get; set; }
    }
}

File: src/ELibrary.Frontend/src/app/modules/shared/domain/models/shop/statistics/getShopStatistics.ts
import { Book } from "../../../..";

export interface GetShopStatistics {
    fromUTC: Date | null,
    toUTC: Date | null,
    includeBooks: Book[]
}

export function getDefaultGetShopStatistics(): GetShopStatistics {
    return {
        fromUTC: null,
        toUTC: null,
        includeBooks: []
    }
}